server/handlers/default.go

```diff
@@ -152,6 +162,7 @@ func DeleteHandler(ctx context.Context, w http.ResponseWriter, r *http.Request)
 	s := ctx.Value("metaStore")
 	store, ok := s.(storage.MetaStore)
 	if !ok {
+		logrus.Error("500 DELETE repository: no storage exists")
```

Comment:
Should this be `logger.Error`? (would need to pull the logger out of the context like other handlers)